app/templates/_configs/with-apollo/packages/project/src/accounts/view/components/accounts-list.component.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import type { AccountsListDataEntity } from '../../domain/models/accounts-list';
+import { AccountsRepository } from "../../data/accounts.repository";
+import { Link, Switch } from 'react-router-dom';
+import { CustomRouteChildrenProps, RouteWithSubRoutes } from '../../../common/view/router/config';
+
+export interface AccountsListProps extends CustomRouteChildrenProps {
+  /**
+   * Get a page
+   */
+  page?: number;
+};
+
+function AccountsList({ page = 1, routes }: AccountsListProps) {
+  const [list, setList] = useState<AccountsListDataEntity[]>([]);
+
+  useEffect(() => {
+    void AccountsRepository.getAccountsList().then((response) => {
+      setList(response.data);
+    });
+    void AccountsRepository.getAccountId('1').then((response) => {
+      // eslint-disable-next-line no-console
+      console.log(response)
+    });
+  }, [page]);
```

Comment:
`page` is not used in this hook, it isn't a dependency

Reply:
Yes this is from old axios implementation as the graphql api does not have pages.